esp/src/eclwatch/nls/hpcc.js

```diff
@@ -344,6 +361,8 @@ define({root:
     Operations: "Operations",
     Options: "Options",
     OriginalFile: "Original File",
+    OrphanFile2: "Orphan File:",
+    OrphanMessage: "An orphan file has partial file parts on disk. However, a full set of parts is not available to construct a complete logical file. These partial file parts, therefore, do not have a reference in the Dali server.",
```

Comment:
"An orphan file has partial file parts on disk. However, a full set of parts is not available to construct a complete logical file. This no reference to these file parts in the Dali server."